tests/docker-compose.yml

```diff
@@ -76,6 +76,22 @@ services:
     volumes:
       - pyesdata7:/usr/share/elasticsearch/data
 
+  elasticsearch8:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.4.0
```

Comment:
Is there a moving tag we can target with `8.x` instead of `8.4` so this stays up to date? I notice above we've been on 7.9.0 for awhile for 7.x testing.

Reply:
No, for some reason, we don't push moving tags to docker hub, see https://hub.docker.com/_/elasticsearch/tags

Reply:
👍